plugins/sep/sli/slipresentation.cc

```diff
@@ -303,17 +305,25 @@ SliPresentation::getPixelAverages(Scroom::Utils::Rectangle<int> area) {
     B = surfaceBegin[offset + 0];
     G = surfaceBegin[offset + 1];
     R = surfaceBegin[offset + 2];
-    // A = surfaceBegin[offset+3]; // don't need this
+    A = surfaceBegin[offset + 3];
 
     c = (255.0 - R);
     m = (255.0 - G);
     y = (255.0 - B);
     k = std::min({c, m, y});
 
-    C += c - k;
-    M += m - k;
-    Y += y - k;
-    K += k;
+    // transparent -> only the white background of Scroom remains visible
+    if (A == 0) {
+      C += 0.0;
+      M += 0.0;
+      Y += 0.0;
+      K += 0.0;
```

Comment:
This doesn't look necessary to me. 
Instead, I would expect only the following case:
``` 
if (A != 0) {
      C += c - k;
      M += m - k;
      Y += y - k;
      K += k;
}
```

Reply:
I definitely agree!